edge-agent/test/Microsoft.Azure.Devices.Edge.Agent.Kubernetes.Test/CombinedKubernetesConfigProviderTest.cs

```diff
@@ -274,23 +298,33 @@ public void EntrypointOptionsWillExist()
             var runtimeInfo = new Mock<IRuntimeInfo<DockerRuntimeConfig>>();
             runtimeInfo.SetupGet(ri => ri.Config).Returns(new DockerRuntimeConfig("1.24", string.Empty));
 
-            var module = new Mock<IModule<DockerConfig>>();
-            module.SetupGet(m => m.Config).Returns(new DockerConfig("nginx:latest", EntryPointCreateOptions, Option.None<NotaryContentTrust>()));
-            module.SetupGet(m => m.Name).Returns("mod1");
+            var module1 = new Mock<IModule<DockerConfig>>();
+            module1.SetupGet(m => m.Config).Returns(new DockerConfig("nginx:latest", EntryPointCreateOptions, Option.None<NotaryContentTrust>()));
```

Comment:
not related here: I am surprised how notary content trust code still exists. this was overwritten with new design long ago :(

Reply:
i am working on it  to find why

Reply:
1.0.10 rc1 was branched off of master which did included the old design changes. i guess we cannot do anything now. so will leave it this way

Reply:
Is there anything that needs to be changed?

Reply:
Option.None<NotaryContentTrust> is obsolete. This PR is needed to offset it. https://github.com/Azure/iotedge/commit/a0971fdcbe790e9c07cb042314408fc2b256aed7